### ISSUE TITLE
Document missing columns for the model sheet in XLSX converter

### ISSUE THREAD

**Issue by wbrunette** [opendatakit/tool-suite-X#153]
Missing the elementType column and itemType column documentation.

**Comment by linl33** [comment]
Also missing `valuesList`

**Comment by victory-lydia** [comment]
Hi I am Victory Excel an Outreachy applicant for the project "New Date Prompts for ODK-X Survey". @elmps2018 I wish to work on this issue

**Comment by victory-lydia** [comment]
Hi,I am Victory Excel an outreachy applicant for the project "New Date Prompts for ODK-X Survey". please @elmps2018 can I work on this issue. And please I will need a little explanation on how to go about it.

**Comment by elmps2018** [comment]
Hi @victory-lydia! I have assigned this issue to you! Since this relates to the documentation, please start by setting up the environment to contribute to the docs (https://docs.odk-x.org/contributing/). You will then need to add to the documentation about XLSX converter (https://docs.odk-x.org/xlsx-converter-using/ or (https://docs.odk-x.org/xlsx-converter-reference/) to explain these columns. Because they are not yet documented, you will probably need to look through the app-designer/xlsx-converter codebase or examples in app-designer to see how they work! Let us know questions while working away!

**Comment by wbrunette** [comment]
@victory-lydia Thank you for taking on this issue! If you search "elementType" on https://forum.odk-x.org or https://docs.odk-x.org you will find references to this, but it is not documented on the "model" sheet description. Like the "survey" sheet description there are optional columns that should be in a second optional table. You will even see elementType is mentioned in the paragraph below. The goal of this issue is to document the optional columns of the model sheet.

**Comment by victory-lydia** [comment]
@elmps2018  Thanks.

**Comment by JudeUk** [comment]
@wbrunette @elmps2018 I will like to take up this issue. Currently looking through the app-designer/xlsx-converter codebase and examples in app-designer to see how the Columns work and what they do. I'll revert if i have any questions

**Comment by elmps2018** [comment]
@JudeUk great I assigned to you! Let us know if you have any questions!

**Comment by JudeUk** [comment]
Hello @elmps2018 i need your help please, following this guide (https://docs.odk-x.org/contributing/), i decided to give Issue 207 — Line Edits a try as a first contribution. I have set up my docs contributing environment locally and even went on to make some modifications on a file. My problem is at the point of serving the site locally, to see my edit on a browser, i get this error when i go to the URL on my browser.





Please see attached screenshots of my browser and terminal thank you.
<img width="1440" alt="Screenshot 2021-10-16 at 16 47 20" src="https://user-images.githubusercontent.com/31444432/137615636-ab647a94-aa18-4998-8dd4-f96b87322bbc.png">
<img width="1440" alt="Screenshot 2021-10-16 at 16 47 37" src="https://user-images.githubusercontent.com/31444432/137615639-73eed5b1-5954-47a3-9c97-c8da2d44086e.png">

**Comment by elmps2018** [comment]
Hi @JudeUk please see comments on the Forum and the other issue you posted! If you can kindly post one place (the Forum is good for support problems that aren't specific to the issue) this saves us all time :)

**Comment by JudeUk** [comment]
> @victory-lydia Thank you for taking on this issue! If you search "elementType" on https://forum.odk-x.org or https://docs.odk-x.org you will find references to this, but it is not documented on the "model" sheet description. Like the "survey" sheet description there are optional columns that should be in a second optional table. You will even see elementType is mentioned in the paragraph below. The goal of this issue is to document the optional columns of the model sheet.

hi @wbrunette Please do you mean  model "worksheet" when you say "sheet description" in your reply? thanks in advance for your kind reply

**Comment by wbrunette** [comment]
@JudeUk  yes the model "worksheet" or "sheet" or "tab" in an XLSX workbook

**Comment by Goodiec** [comment]
Hi @wbrunette. @elmps2018. Can I work on this issue?

**Comment by wbrunette** [comment]
@Goodiec if you know what the elementType and itemType columns are go ahead and document them (you can also research to find the answer).
